pubspec.yaml

```
@@ -9,23 +9,31 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
+  cupertino_icons: ^1.0.2
   flutter:
     sdk: flutter
-  cupertino_icons: ^1.0.2
+  flutter_native_splash: ^2.2.17
 
   # State Management
   stacked: ^3.1.0-beta.0
   stacked_services: ^0.9.8
 
 dev_dependencies:
-  flutter_test:
-    sdk: flutter
+  build_runner: ^2.2.0
 
   flutter_lints: ^1.0.0
-  build_runner: ^2.2.0
+  flutter_test:
+    sdk: flutter
+  mockito: ^5.1.0
 
   stacked_generator: ^0.8.1
-  mockito: ^5.1.0
 
 flutter:
   uses-material-design: true
+  assets:
+    - assets/png/
```

I will prefer we have `images`, `svgs`, etc.. So this can be 
```suggestion
  assets:
    - assets/images/
```

---

Done